state.go

```diff
@@ -388,6 +341,39 @@ func (o *OverflowState) Address(key string) string {
 	return fmt.Sprintf("0x%s", o.FlowAddress(key))
 }
 
+// return the flow Address of the given name
+func (o *OverflowState) FlowAddressE(key string) (*flow.Address, error) {
+	account, err := o.AccountE(key)
+	if err == nil {
+		return &account.Address, nil
+	}
+
+	flowContract, err := o.State.Contracts().ByName(key)
+	if err != nil {
+		return nil, err
+	}
+
+	// we found the contract specified in contracts section
+	if flowContract != nil {
+		alias := flowContract.Aliases.ByNetwork(o.Network.Name)
+		if alias != nil {
+			return &alias.Address, nil
+		}
+	}
+
+	flowDeploymentContracts, err := o.State.DeploymentContractsByNetwork(o.Network)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, flowDeploymentContract := range flowDeploymentContracts {
+		if flowDeploymentContract.Name == key {
+			return &flowDeploymentContract.AccountAddress, nil
+		}
+	}
+	return nil, fmt.Errorf("Not valid user account, contract or deployment contract")
```

Comment:
could use `errors.New`

Reply:
Can you try to use the suggest feature to add this comment so that the overhead of fixing it is not high?